caas/device.mk

```diff
@@ -419,17 +419,17 @@ PRODUCT_PACKAGES += \
     hwcomposer.drm_minigbm
```

Comment:
most of files are update automatically using mixin_update, you should not change it directly